src/main/app/case/CosApiClient.ts

```diff
@@ -339,6 +339,61 @@ export class CosApiClient {
     });
     return response.data;
   }
+
+  public async retrieveCaseHearingsByCaseId(caseId: string, user: UserDetails): Promise<CaseWithId> {
+    if (!caseId || !user) {
+      return Promise.reject(new Error('retrieveCaseHearingsByCaseId - Case id must be set and user must be set'));
+    }
+    // const response = await Axios.get(config.get('services.cos.url') + `/${caseId}`, {
+    //   headers: {
+    //     Authorization: 'Bearer ' + user.accessToken,
+    //     serviceAuthorization: getServiceAuthToken(),
+    //     Accept: 'application/json',
+    //     'Content-Type': 'application/json',
+    //   },
+    // });
+
+    //req.session.userCase = {id:'1234', state: State.Holding};
+    //req.session.userCase.hearingCollection = [];
```

Comment:
can we remove this code smells please.

Reply:
will remove this commented code, Once the backend api is ready for /hearings we can make axios call to get the hearings data